tests/test_api_routes.py

```diff
@@ -0,0 +1,108 @@
+import pytest
+from data.duplicates import STAMPY_DUPLICATES
+
+
+def test_duplicates(client, stampy_duplicates):
+    response = client.get('/api/duplicates')
+    assert response.status_code == 200
+    assert response.json == STAMPY_DUPLICATES
+
+
+def test_search_defaults(client, mock_retriever_model):
```

Comment:
test the various search options. Let me know if more (or different options) should be added

Reply:
will do